datasource.go

```diff
@@ -0,0 +1,205 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"golang.org/x/net/context"
+
+	"github.com/bitly/go-simplejson"
+	"github.com/grafana/grafana-plugin-model/go/datasource"
+	plugin "github.com/hashicorp/go-plugin"
+	"golang.org/x/net/context/ctxhttp"
```

Comment:
Can this be removed due to line 13? And access changed to `context.ctxhttp.Do`?

Reply:
No, this cannot be removed since `ctxhttp` is a subpackage of `context`. It is not exported so I can't just reference `context.ctxhttp`